blender/arm/props_ui.py

```diff
@@ -1423,7 +1529,24 @@ def draw(self, context):
         col.prop(rpdat, 'arm_pcfsize')
         layout.separator()
 
+        def label_multiline(text: str, context: bpy.types.Context, parent):
+            textListSplitByNewLines = text.split('\n')
+            import textwrap
+            for text in textListSplitByNewLines:
+                chars = int(context.region.width / 7)   # 7 pix on 1 character
+                wrapper = textwrap.TextWrapper(width=chars)
+                text_lines = wrapper.wrap(text=text)
+                for text_line in text_lines:
+                    parent.label(text=text_line)
         layout.prop(rpdat, 'rp_shadowmap_atlas')
+        if (not rpdat.rp_shadowmap_atlas) and os.name == 'nt':
+            EditorTimeWarnings.registerWarning("Armory Render Path > Shadows > Shadow Map Atlasing", "Known to prevent Armory from starting on Windows when there are more than 2 scene lights.", True, context, layout)
```

Comment:
This sounds as if the Shadow Map Atlas option would prevent Armory from compiling whereas the opposite is the case – enabling this option is a workaround to fix shader compilation on Direct X (which may be better wording than "Windows" since it runs perfectly fine when targeting html5 on Windows for example, it's not the OS that's the problem).

I really hope that we'll find that nasty shader compilation bug soon so that we can remove this warning again.

Reply:
Ah yes, typo, fixing.